pkg/providers/docker/validator_test.go

```diff
@@ -0,0 +1,30 @@
+package docker
```

Comment:
Can we put this file in the `docker_test` package? We can make the new method public so that we can call it from here